presto-main/src/test/java/com/facebook/presto/block/TestDictionaryBlock.java

```diff
@@ -140,6 +144,38 @@ public void testCompact()
     }
 
     @Test
+    public void testCompactBlocks()
+            throws Exception
+    {
+        DictionaryId commonSourceId = randomDictionaryId();
+
+        Slice[] expectedValues = createExpectedValues(5);
+        // Both blocks do not reference value 2, only block2 references value 4
+        DictionaryBlock dictionaryBlock1 = createDictionaryBlockWithUnreferencedKeys(expectedValues, 10, commonSourceId);
+        DictionaryBlock dictionaryBlock2 = createDictionaryBlock(expectedValues, new int[] {0, 1, 3, 4, 1, 4, 3, 0}, commonSourceId);
+
+        assertEquals(dictionaryBlock1.isCompact(), false);
+        assertEquals(dictionaryBlock2.isCompact(), false);
+        List<Block> compactBlocks = DictionaryBlock.compactBlocks(ImmutableList.of(dictionaryBlock1, dictionaryBlock2));
+
+        DictionaryBlock compactBlock;
+        // first block
+        compactBlock = (DictionaryBlock) compactBlocks.get(0);
```

Comment:
I think this test would be easier to read if you renamed `compactBlock` to `compactBlock1` and `compactBlock2`